manifests/init.pp

```diff
@@ -71,6 +71,24 @@
         notify  => Service[$servicename],
         content => template('dhcp/redhat/sysconfig-dhcpd'),
```

Comment:
Sorry, I'm not sure I follow.  Isn't the point of this that the systemd unit needs to be edited, not the sysconfig file?  What does the sysconfig file do?